packages/backend-openapi-utils/README.md

```diff
@@ -61,6 +61,12 @@ export function createRouter() {
 }
 ```
 
+## FAQs
+
+### Why am I getting `unknown` as the type for a response?
+
+This can happen when you have a `charset` defined in your `response.content` section. Something like `response.content[    'application/json; charset=utf-8:']` will cause this issue.
```

Comment:
```suggestion
This can happen when you have a `charset` defined in your `response.content` section. Something like `response.content['application/json; charset=utf-8:']` will cause this issue.
```